### PR TITLE
#163591958 A following count should return for a given profile

### PR DESCRIPTION
**Pull request by d-rita**
#### What does this PR do?
This PR fixes the logic for returning the count of author profiles a user is following as well as their followers. 

#### Description of task to be completed
Retrieve the length of the lists of followers and people a user is following in their profiles. 

#### How should this be manually tested?
- Fetch this branch in your terminal and check out into it using `git fetch origin bg-following-count-163591958 && git checkout bg-following-count-163591958`
- Start the application by running the server with `python manage.py runserver`
- To test the endpoint: 
   - first, follow a user using `POST/api/profiles/<username-to-follow>/follow`
   - then, check both the follower and followed users' profiles to see their following counts by using `GET/api/profiles/<username>`

#### Screenshots
screen shot 2019-01-30 at 6 47 29 pm
screen shot 2019-01-30 at 6 48 37 pm

#### Related Pivotal Tracker Story
[#163591958](https://www.pivotaltracker.com/story/show/163591958)

### CODE REVIEW

File: authors/apps/profiles/serializers.py [hunk 1]
```diff
@@ -36,10 +36,9 @@ def get_followers(self,obj):
             obj(instance): This is a profile instance of a user
 
         """
-        followers = []
-        for user in list(obj.followers.all()):
-            followers.append(user.username)
-        return len(followers)
+        return len(list(obj.followers.all()))
```
Comment (armstrongsouljah) [inline authors/apps/profiles/serializers.py:39]: Did you try `obj.followers.count()`

**Comment by MuhweziDeo** [comment]
@d-rita  is this the bug fix

**Comment by d-rita** [comment]
@MuhweziDeo yes it is. Before it would return the following count for a logged in user regardless of whose profile you visited.

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by mariamiah — APPROVED** [review]
LGTM

File: authors/apps/profiles/serializers.py (hunk 1 above)
Comment (d-rita) [reply authors/apps/profiles/serializers.py:39]: It had skipped my mind. Thanks

**Review by calvinpete — APPROVED** [review]
LGTM

**Review by MuhweziDeo — APPROVED** [review]

**Pull request merged**